apps/web/app/utils/post-link.ts

```diff
@@ -1,6 +1,17 @@
-export function getPostHref(slug: string) {
-  if (process.env.NODE_ENV === 'development') {
-    return `https://prosperous-combination-099461.framer.app/posts/${slug}`;
+export function getPostHref(slug?: string) {
+  if (slug === undefined || slug === null) {
+    slug = 'undefined';
```

Comment:
This seems like pointless code? Don't we just want to return and do nothing?

Reply:
The test expects this behavior - it specifically checks that undefined slugs return a URL with "/posts/undefined". Guess we can change that, lmk what you think